app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -56,11 +55,37 @@ class SearchLocationFragment : Fragment() {
             redirectToMain()
         })
 
-        setupPlaceAutoCompleteFrag(savedInstanceState)
+        rootView.locationSearchView.setOnQueryTextListener(object : SearchView.OnQueryTextListener {
```

Comment:
What'll happen here?

Reply:
In short, I reverted it back to old SearchLocationFragment where there is only a searchview for user to type in the name of the the place, there isn't location suggestion anymore (I will improve this fragment later). Here is the view:
<img src="https://scontent-ams3-1.xx.fbcdn.net/v/t1.15752-9/55887679_2373451552677902_3202881091153166336_n.png?_nc_cat=106&_nc_ht=scontent-ams3-1.xx&oh=1bba92bc4c1d2e3a2e82ffc5554f463a&oe=5D504E60" width="300">